backend/strawberry_forms/converter.py

```diff
@@ -0,0 +1,120 @@
+@convert_form_field.register(forms.Field)
+def convert_form_field_to_string(field):
+    return (
+        type_or_optional_wrapped(str, field.required),
+        strawberry.field(description=field.help_text, is_input=True),
+    )
+
+
+@convert_form_field.register(forms.UUIDField)
+def convert_form_field_to_uuid(field):
+    return (
+        type_or_optional_wrapped(str, field.required),
+        strawberry.field(description=field.help_text, is_input=True),
+    )
+
+
+@convert_form_field.register(forms.IntegerField)
+@convert_form_field.register(forms.NumberInput)
+def convert_form_field_to_int(field):
+    return (
+        type_or_optional_wrapped(int, field.required),
+        strawberry.field(description=field.help_text, is_input=True),
+    )
+
+
+@convert_form_field.register(forms.BooleanField)
+def convert_form_field_to_boolean(field):
+    return (bool, strawberry.field(description=field.help_text, is_input=True))
+
+
+@convert_form_field.register(forms.NullBooleanField)
+def convert_form_field_to_nullboolean(field):
+    return (
+        Optional[bool],
+        strawberry.field(description=field.help_text, is_input=True),
+    )
+
+
+@convert_form_field.register(forms.DecimalField)
+@convert_form_field.register(forms.FloatField)
+def convert_form_field_to_float(field):
+    return (
+        type_or_optional_wrapped(float, field.required),
+        strawberry.field(description=field.help_text, is_input=True),
+    )
+
+
+@convert_form_field.register(forms.ModelMultipleChoiceField)
+def convert_form_field_to_list(field):
+    return (
+        type_or_optional_wrapped(List[strawberry.ID], field.required),
+        strawberry.field(description=field.help_text, is_input=True),
+    )
+
+
+@convert_form_field.register(forms.DateField)
+def convert_form_field_to_date(field):
+    return (
+        type_or_optional_wrapped(str, field.required),
+        strawberry.field(description=field.help_text, is_input=True),
+    )
+    # return Date(description=field.help_text, required=field.required)
+
+
+@convert_form_field.register(forms.DateTimeField)
+def convert_form_field_to_datetime(field):
+    return (
+        type_or_optional_wrapped(DateTime, field.required),
+        strawberry.field(description=field.help_text, is_input=True),
+    )
+    # return DateTime(description=field.help_text, required=field.required)
+
+
+@convert_form_field.register(forms.TimeField)
+def convert_form_field_to_time(field):
+    return (
+        type_or_optional_wrapped(str, field.required),
+        strawberry.field(description=field.help_text, is_input=True),
+    )
+    # return Time(description=field.help_text, required=field.required)
+
+
+@convert_form_field.register(forms.ModelChoiceField)
+def convert_form_field_to_id(field):
+    return (
+        type_or_optional_wrapped(strawberry.ID, field.required),
+        strawberry.field(description=field.help_text, is_input=True),
+    )
+    # return ID(required=field.required)
```

Comment:
Well done! Maybe we can remove the comments now 😄

Reply:
Done!